pkg/kritis/secrets/secrets.go

```diff
@@ -48,17 +59,31 @@ func Fetch(namespace string, name string) (*PGPSigningSecret, error) {
 	if err != nil {
 		return nil, err
 	}
-	pub, ok := secret.Data[constants.PublicKey]
+	pub, ok := secret.Data[PublicKey]
 	if !ok {
-		return nil, fmt.Errorf("invalid secret %s. could not find key %s", name, constants.PublicKey)
+		return nil, fmt.Errorf("invalid secret %s. could not find key %s", name, PublicKey)
 	}
-	priv, ok := secret.Data[constants.PrivateKey]
+	priv, ok := secret.Data[PrivateKey]
 	if !ok {
-		return nil, fmt.Errorf("invalid secret %s. could not find key %s", name, constants.PublicKey)
+		return nil, fmt.Errorf("invalid secret %s. could not find key %s", name, PrivateKey)
+	}
+	pb, ok := secret.Data[Passphrase]
+	phrase := ""
```

Comment:
What's the expected behavior if !ok?

Reply:
If !ok then passphrase wasn't provided, which is acceptable. If passphrase was needed to decrypt the private key and it wasn't set, then we'll fail to do verification.